table/block_based/filter_policy.cc

```diff
@@ -259,9 +270,9 @@ class XXPH3FilterBitsBuilder : public BuiltinFilterBitsBuilder {
   std::shared_ptr<const FilterPolicy> filter_policy_;
 
   struct HashEntriesInfo {
-    // A deque avoids unnecessary copying of already-saved values
-    // and has near-minimal peak memory use.
-    std::deque<uint64_t> entries;
+    // A list avoids unnecessary copying of already-saved values
+    // and has less peak memory use than vector.
```

Comment:
Question: 
Is the "near-minimal peak memory use." referring to the underlying impl of std::deque not pre-allocating memory slots as std::vector? I reworded for the list case.

Reply:
Please ignore